appengine/firebase-tictactoe/src/test/java/com/example/appengine/firetactoe/MoveServletTest.java

```diff
@@ -136,12 +136,13 @@ public LowLevelHttpResponse execute() throws IOException {
         };
       }
     });
-    FirebaseChannel.getInstance().httpTransport = mockHttpTransport;
+    FirebaseChannel.getInstance(null).httpTransport = mockHttpTransport;
 
+    Mockito.doReturn(null).when(servletUnderTest).getServletContext();
     servletUnderTest.doPost(mockRequest, mockResponse);
 
     game = ofy.load().type(Game.class).id(gameKey).safe();
-    assertThat(game.board).isEqualTo(" X       ");
+    assertEquals(game.board, " X       ");
```

Comment:
Truth might make this cleaner.

Reply:
Minimizing test dependencies, so leaving this as-is